mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp

```diff
@@ -1016,8 +1017,47 @@ int CNEOBaseCombatWeapon::DrawModel(int flags)
 	C_BaseCombatCharacter* localPlayer = C_BasePlayer::GetLocalPlayer();
 	if (GetOwner() == localPlayer && ShouldDrawLocalPlayerViewModel())
 		return 0;
+	
+	auto ret = BaseClass::DrawModel(flags);
 
-	return BaseClass::DrawModel(flags);
+	auto owner = dynamic_cast<C_NEO_Player*>(GetOwner());
```

Comment:
we're using a lot of static_cast in many other places, I think this is a safe assumption now.